packages/x-date-pickers/src/themeAugmentation/props.d.ts

```diff
@@ -31,16 +31,17 @@ import {
   PickersToolbarProps,
   PickerStaticWrapperProps,
   ExportedCalendarHeaderProps,
+  CalendarOrClockPickerView,
 } from '../internals';
```

Comment:
It does not seem to be exported from `internals`.

```suggestion
} from '../internals';
import { CalendarOrClockPickerView } from '../internals/models';
```

Reply:
`MuiLocalizationProvider` also requires a generic.
Should probably be:
```diff
-MuiLocalizationProvider: LocalizationProviderProps;
+MuiLocalizationProvider: LocalizationProviderProps<unknown>;
```